introduction/hardware-options/required-hardware/camera.md

```diff
@@ -0,0 +1,35 @@
+# Camera
+
+The Companion Computer software contains a series of drivers which will stream HD video from the vehicle to the Ground Control Station (GCS) at the surface. Only one camera may be connected to the Companion at a time. Support for multiple camera streams is planned, but not implemented at this time.
+
+# Supported Cameras
+
+## USB Cameras
+
+Most modern USB cameras are compatible with the Companion software as long as they have the following specifications:
+
+* H.264 Output
+* 1080p resolution (4k cameras are not supported)
```

Comment:
4k cameras should be supported in the last QGC release.

Reply:
@patrickelectric A USB 4K camera plugged into the Pi will work?

I know the 4K Ethernet ones do, but wasn't sure about a USB one.

Reply:
Text Corrected.